eos/models/wilson-coefficients.hh

```diff
@@ -168,23 +168,37 @@ namespace eos
          * These two operators are defined via
          *
          * H^eff = 4 G_F / sqrt(2) V_tb V_ts^* (
-         *             C_L alpha_e / (2 pi) * [sbar gamma^mu P_L b] [nubar gamma_mu P_L nu]
-         *             C_R alpha_e / (2 pi) * [sbar gamma^mu P_R b] [nubar gamma_mu P_L nu]
+         *             C_L alpha_e / (2 pi) * [nubar gamma_mu P_L nu] [sbar gamma^mu P_L b] 
+         *             C_R alpha_e / (2 pi) * [nubar gamma_mu P_L nu] [sbar gamma^mu P_R b] 
          *         )
-         *       = 4 G_F / sqrt(2) V_tb V_ts^* ( C_L O_L + C_R O_R )
+         *       = 4 G_F / sqrt(2) V_tb V_ts^* ( C_VLL O_VLL + C_VLR O_VLR )
+         * 
+         * Furthermore, we can add the scalar modes [FLS:2021A], eq. (2)
+         * O_SLL = [bar(nu_L^c) P_L nu][sbar P_L b], where nu_L^c = C bar(nu_L)^T, C = i gamma^2 gamma^0
```

Comment:
Your nu_L is incorrect, is it not? ``nu_L^c = (C nubar_L)^T`` ?

Reply:
I think it is correct, at least according to [FLS:2021A] page 4, second paragraph. Since C is a matrix and nubar_L is a row vector, so I should transpose it before applying C, if I am not mistaken. Please correct me if I'm wrong (:

Reply:
You are entirely correct, I had a knot in my brain.